arangod/Cluster/Maintenance.cpp

```diff
@@ -1865,6 +1867,9 @@ arangodb::Result arangodb::maintenance::reportInCurrent(
           auto result = replication::parseVersion(rv);
           TRI_ASSERT(result.ok());
           replicationVersion = std::move(result.get());
+        } else {
+          // if "replicationVersion" field is not found this must be an old DB
+          // which defaults to version ONE.
```

Comment:
Is this intentional (no code just else with comment)?

Reply:
Yes, the variable is already initialized with version::ONE in the definition. However, I wanted to place the comment here were we actually have the check instead of at the definition.